xcodeproj/internal/input_files.bzl

```diff
@@ -775,7 +775,7 @@ def _from_resource_bundle(bundle):
         non_arc_srcs = EMPTY_LIST,
         pch = None,
         resource_bundle_dependencies = bundle.dependencies,
-        resources = depset(bundle.resources),
+        resources = depset([(resource, None) for resource in bundle.resources]) if bundle.resources else EMPTY_DEPSET,
```

Comment:
I think you will need to do `folder_resources` as well, which are just structured resources.